modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/manager/v1_0/DefaultObjectEntryManagerImpl.java

```diff
@@ -1384,11 +1385,16 @@ private void _processAttachment(
 		FileEntry fileEntry = ObjectMapperUtil.readValue(
 			FileEntry.class, propertyValue);
 
-		if ((fileEntry == null) || (fileEntry.getFileBase64() == null)) {
+		if ((fileEntry == null) ||
+			((fileEntry.getFileBase64() == null) &&
+			 (fileEntry.getExternalReferenceCode() == null))) {
+
 			return;
 		}
 
-		if (fileEntry.getId() != null) {
+		if ((fileEntry.getId() != null) &&
+			(fileEntry.getFileBase64() != null)) {
```

Comment:
should we also reorder here?

Reply:
I ended up removing this exception in this [commit](https://github.com/liferay-objects/liferay-portal/pull/3617/commits/7e7bfb22f4abfadaef42d0e3c2d893fbaa8f8c2b). Sorry for the confusion.